config/crd/bases/elbv2.k8s.aws_ingressclassparams.yaml

```diff
@@ -106,17 +125,24 @@ spec:
                   matchLabels:
                     additionalProperties:
                       type: string
-                    description: matchLabels is a map of {key,value} pairs. A single {key,value} in the matchLabels map is equivalent to an element of matchExpressions, whose key field is "key", the operator is "In", and the values array contains only "value". The requirements are ANDed.
+                    description: matchLabels is a map of {key,value} pairs. A single
+                      {key,value} in the matchLabels map is equivalent to an element
+                      of matchExpressions, whose key field is "key", the operator
+                      is "In", and the values array contains only "value". The requirements
+                      are ANDed.
                     type: object
                 type: object
+                x-kubernetes-map-type: atomic
```

Comment:
Can I ask what purpose it serves to add this into the template? Is it a dependency of something? I don't see anything in the README about it.